frontend/pages/investors/new.tsx

```diff
@@ -62,132 +116,66 @@ const NewInvestorPage: PageComponent<{}, FormPageLayoutProps> = (props) => {
 
       <MultiPageLayout
         layout="narrow"
-        title={intl.formatMessage({ defaultMessage: 'Setup investor profile', id: '7Rh11y' })}
+        title={formatMessage({ defaultMessage: 'Setup project developer’s account', id: 'bhxvPM' })}
```

Comment:
This title is wrong.

```suggestion
        title={formatMessage({ defaultMessage: 'Setup investor profile', id: '7Rh11y' })}
```